### ISSUE TITLE
Allow variable list to be reverse order, or a quick "new variable" option

### ISSUE THREAD

**Issue by dogpartyguy** [chrismaltby/gb-studio#469]
With the length it can reach now on advanced games, scrolling through the variable list in a small window at the bottom corner of the screen can be more than a bit fiddly and awkward, and the issue only grows over time. Why not allow some sort of easy new variable option or a "reverse order" (most recently added, to earliest, or such) to make things quicker and easier in large projects?

**Comment by the1andOnlyANTI** [comment]
I agree, any tools to help sort and organize the variable list would be helpful.

**Comment by RichardULZ** [comment]
Note that you can already search for variables by typing, if you know the number, or name, both should work already.

**Comment by the1andOnlyANTI** [comment]
> Note that you can already search for variables by typing, if you know the number, or name, both should work already.

Yeah you can select a variable from the drop down variable list by doing that, but I'm talking about searching for a variable or event name to find any different places within the actor or scene's script where that variable is used.

When several actors and scenes have events to modify a certain variable it would be nice to have an easy way to find where that variable has been used.

**Comment by dogpartyguy** [comment]
> 
> 
> Note that you can already search for variables by typing, if you know the number, or name, both should work already.

But when you have upward of 50+ variables spread across potentially a hundred or more scenes, remembering them all or even just the last dozen or half dozen can be a little tricky.  And creating a new one, which this would alleviate, even more so if you don't remember the number or names already used.
